templates/.VERSION

```diff
@@ -1 +1 @@
-0.9.99.0915
\ No newline at end of file
+v0.9.0+dev
```

Comment:
@lunny what I'm saying is not related to semver format but to reality. We tagged `v0.9.99`, so current template .VERSION cannot be in the past, and `v0.9.0+dev` is in the past, no ?